From a6cbafc6b4c6b4d1db13006008bda1c3152bbf58 Mon Sep 17 00:00:00 2001
From: John Ferlan <jferlan@redhat.com>
Date: Wed, 22 Jan 2014 13:13:02 -0500
Subject: [PATCH 43/60] libxkutil/xml_parse_test: Coverity cleanup

A new version of Coverity found:

print_domxml(): RESOURCE_LEAK
  - The 'xml' variable needs to be free()'d

Signed-off-by: John Ferlan <jferlan@redhat.com>
---
 libxkutil/xml_parse_test.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libxkutil/xml_parse_test.c b/libxkutil/xml_parse_test.c
index 374bcf6..303880d 100644
--- a/libxkutil/xml_parse_test.c
+++ b/libxkutil/xml_parse_test.c
@@ -192,6 +192,7 @@ static void print_domxml(struct domain *dominfo,
                 printf("Failed to create system XML\n");
         else
                 printf("%s\n", xml);
+        free(xml);
 }
 
 static char *read_from_file(FILE *file)
-- 
2.1.0

